WrightTools/artists.py

```diff
@@ -2195,16 +2195,9 @@ def plot(self, channel=0,
             for xi, yi, kwargs in self._onplotdata:
                 subplot_main.plot(xi, yi, **kwargs)
             # colorbar ----------------------------------------------------------------------------
-            subplot_cb = plt.subplot(gs[1])
+            cax = plt.subplot(gs[1])
             cbar_ticks = np.linspace(levels.min(), levels.max(), 11)
-            if cbar_ticks.max() == 1.0:
```

Comment:
I'm not sure I like this particular way of determining it, but do we wish to maintain single decimal place for normalized data?

I think it may be something we consider doing in a smarter way in general, not sure exactly how, but I feel that there should exist such a smart detection algorithm, that also scales to different orders of magnitude.

Unclear to me whether this detection should be moved to plot_colorbar or not

Other than that, it seems to function as expected from my limited testing.